x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/lib/utils.ts

```diff
@@ -103,7 +103,7 @@ const replaceAliasPathByAliasId = (
   Object.entries(byId).forEach(([id, field]) => {
     if (field.source.type === 'alias') {
       const aliasTargetField = Object.values(byId).find(
```

Comment:
nit: just saw this now, but `Object.values(byId)` can be calculated once outside of the `.forEach`.

Reply:
This is actually not a nit, it would be a NNNIIITTT if that word existed 😄 It caused a big performance issue that I saw and fixed in https://github.com/elastic/kibana/pull/54224

Great catch though! 👍